src/main/java/io/openliberty/tools/intellij/util/LibertyProjectUtil.java

```diff
@@ -113,6 +117,16 @@ private static ArrayList<BuildFile> getBuildFiles(Project project, String buildF
         return buildFiles;
     }
 
+    /**
+     * 
+     * @param buildFile maven or gradle build file in the form of PsiFile
+     * @return <code>true</code> if the project contains src/main/liberty/config/server.xml relative to the build file; <code>false</code> otherwise
+     */
+    private static boolean isLibertyProject(PsiFile buildFile) {
+        String rootDir = buildFile.getVirtualFile().getParent().getCanonicalPath();
+        return new File(rootDir, "src/main/liberty/config/server.xml").exists();
```

Comment:
I'm worried about this potentially not working properly in Windows. Will try to test and modify if needed.

Reply:
Works in Windows